integration-tests/src/test/rpc/creditcoin.test.ts

```diff
@@ -26,7 +26,7 @@ describe('Creditcoin RPC', (): void => {
                     getOffchainNonceKey: {
                         params: [
                             {
-                                name: 'acc',
+                                name: 'account_id',
```

Comment:
Rename is fine but the commit log message doesn't match the reality. Probably a leftover from before.